webapp/channels/src/components/widgets/users/avatar/avatar.tsx

```diff
@@ -27,6 +27,10 @@ export const getAvatarWidth = (size: TAvatarSizeToken) => {
         return 36;
     case 'xl':
         return 50;
+    case 'xl-custom-DM':
+        return 96;
+    case 'xl-custom-GM':
+        return 72;
```

Comment:
Are we sure we want to add extra sizes for the Avatar widget? It's not a big deal, but I just want to make sure since I don't really like the look of having more niche overrides like this

Reply:
@hmhealey We do need these sizes, but I was on the fence about how to go about doing it. I'd rather add in other token sizes that follow the existing convention, but I would have to change what the existing names mean likely to do that. For example, I'd make the current `xxs` become `xxxs` and `xxl` become `xxxl` to make room for the two new sizes in between.

If you'd rather I do that, I can, but it will mean a lot of other file changes.

Reply:
Actually, have a look at the changes I made. Only a few files needed to be updated to make these more general and not so niche/overidey (not a word).